Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
Didn't use Argo before, Looks nice. I think we can consider using Argo when we'll have more than one thing to decode. Now this is only `DailyForecast` which has very simple structure and IMO integrating additional tool to decode it is a little bit overkill :) What you think?